### ISSUE TITLE
Table styling

### ISSUE THREAD

**Issue by rufuspollock** [okfn/jekyll-template#1]

**Comment by rufuspollock** [comment]
@mintcanary love your help here - see http://fiscal.dataprotocols.org/spec/ as example of not great rendering

**Comment by rufuspollock** [comment]
@mintcanary let me know if you get a chance to fix this (or may be we obsolete by merging the open data handbook styling)

**Comment by smth** [comment]
As this is using Bootstrap, an easy solution would be to add a class of `table` (and any [other class](http://getbootstrap.com/css/#tables) you feel appropriate) to the tables. Not an ideal solution, but probably the best one given the use of a CDN.

**Comment by rufuspollock** [comment]
@mintcanary good point - though it may also be nice to copy into our over-riding style.css default styling for tables (useful in cases where you cannot set css easily).
